### PR TITLE
PIM-3806: delete an attribute from a product template

### PR DESCRIPTION
**Pull request by skeleton**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | Y
| CI currently passes? | ?
| Tests pass?          | Y
| Scenarios pass?      | Y
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | Y
| Fixed tickets        | PIM-3806
| DB schema updated?   | N
| Migration script?    | N
| Doc PR               | N

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +35,7 @@
 - PIM-3753: Fix completeness filter
 
 ## BC breaks
+- Change the constructor of `Pim/Bundle/CatalogBundle/Doctrine/Common/Remover/AttributeRemover' to accept `Pim/Bundle/CatalogBundle/Builder/ProductTemplateBuilder` as the fourth argument and accept `Pim/Bundle/CatalogBundle/Entity/Repository/ProductTemplateRepository` as the fifth argument
```
Comment (solivier) [inline CHANGELOG.md]: use ` instead of '

Comment (skeleton) [inline CHANGELOG.md]: Why ? I do not replace service.

File: src/Pim/Bundle/CatalogBundle/Doctrine/Common/Remover/AttributeRemover.php [hunk 2]
```diff
@@ -29,19 +31,31 @@ class AttributeRemover implements RemoverInterface
     /** @var EventDispatcherInterface */
     protected $eventDispatcher;
 
+    /** @var ProductTemplateBuilder */
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Doctrine/Common/Remover/AttributeRemover.php]: hell no, we have interface :)

File: src/Pim/Bundle/CatalogBundle/Doctrine/Common/Remover/AttributeRemover.php [hunk 3]
```diff
@@ -29,19 +31,31 @@ class AttributeRemover implements RemoverInterface
...
+    protected $productTemplateBuilder;
+
+    /** @var ProductTemplateRepository */
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Doctrine/Common/Remover/AttributeRemover.php]: same here :)

File: CHANGELOG.md (hunk 1 above)
Comment (solivier) [inline CHANGELOG.md]: I just speak about the quotes you do not use the good quotes at the end of the line you have to use the ` quote

**Pull request merged**